projects/elife.yaml

```diff
@@ -551,6 +552,7 @@ journal:
                     fallbacks:
                         4xx: "4xx.html"
                         5xx: "5xx.html"
+                ip-blacklist: true # maintained at https://manage.fastly.com/configure/services/1WzbLJKoGqzOF1IrhTNx2M/versions/19/data/acls
```

Comment:
These URIs contain version numbers, but the ACL content isn't part of the version so I think this will still be usable in the future (otherwise you'd have to switch to whatever the active version number is).